airflow/migrations/versions/13bd1cac1ff7_add_run_as_user_column_for_tasks.py

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Add run_as_user column for tasks
+
+Revision ID: 13bd1cac1ff7
+Revises: 8504051e801b
+Create Date: 2016-12-08 17:37:49.417313
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '13bd1cac1ff7'
+down_revision = '8504051e801b'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('task_instance', sa.Column('run_as_user', sa.String(length=1000),
```

Comment:
I'm not sure why this was written to the DB either. Good catch, removing the migration and field from the TaskInstance model.